bridges/zombienet/environments/rococo-westend/spawn.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -e
+
+trap "trap - SIGTERM && kill -9 -$$" SIGINT SIGTERM EXIT
+
+source "${BASH_SOURCE%/*}/../../utils/common.sh"
+source "${BASH_SOURCE%/*}/../../utils/zombienet.sh"
+
+logs_dir=$TEST_DIR/logs
+helper_script="${BASH_SOURCE%/*}/helper.sh"
+
+rococo_def=$POLKADOT_SDK_PATH/cumulus/zombienet/bridge-hubs/bridge_hub_rococo_local_network.toml
+start_zombienet $TEST_DIR $rococo_def rococo_dir rococo_pid
+echo
+
+rococo_init_log=$logs_dir/rococo-init.log
+echo -e "Setting up the rococo side of the bridge. Logs available at: $rococo_init_log\n"
+$helper_script init-asset-hub-rococo-local >> $rococo_init_log 2>&1
+$helper_script init-bridge-hub-rococo-local >> $rococo_init_log 2>&1
+echo
+
+westend_def=$POLKADOT_SDK_PATH/cumulus/zombienet/bridge-hubs/bridge_hub_westend_local_network.toml
+start_zombienet $TEST_DIR $westend_def westend_dir westend_pid
+echo
+
+westend_init_log=$logs_dir/westend-init.log
+echo -e "Setting up the westend side of the bridge. Logs available at: $westend_init_log\n"
+$helper_script init-asset-hub-westend-local >> $westend_init_log 2>&1
+$helper_script init-bridge-hub-westend-local >> $westend_init_log 2>&1
+echo
+
+relay_log=$logs_dir/relay.log
```

Comment:
Also makes sense to have this modular, but I would keep it as part of the environment as well. Not just for the zombienet tests, but even if we just want to spawn the environment and run some tests manually, would be nice to have the possibility of also start the relayer. For the moment didn't modify this, but in the future we can add a flag like `--init` for this too if necessary. But so far I think that we need the relayer to start for all the tests that we have. WDYT ?

Reply:
> But so far I think that we need the relayer to start for all the tests that we have.

Yes, we need to start relayers for all our tests. But OTOH we need to start it at different points - in test1 we start it early. In test2 and test3 it needs to be started later, after at least one session is completed at relay chain. But since you're only modifying first test here, it is ok to leave it for a follow-up PR